ghost/core/core/server/data/migrations/versions/5.19/2022-10-10-06-58-add-subscriptions-table.js

```diff
@@ -0,0 +1,31 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('subscriptions', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    type: {
+        type: 'string', maxlength: 50, nullable: false, validations: {
+            isIn: [['free', 'comped', 'paid']]
+        }
+    },
+    status: {
+        type: 'string', maxlength: 50, nullable: false, validations: {
+            isIn: [['active', 'expired', 'cancelled']]
```

Comment:
It's fine to leave but you don't need `validations` in the migration as they're only used by model layer